sopel/web.py

```diff
@@ -3,7 +3,7 @@
 *Availability: 3+, depreacted in 6.2.0*
```

Comment:
You fixed the extra space below, but not this typo? :joy:

```suggestion
*Availability: 3+, deprecated in 6.2.0*
```

Reply:
Like I've seen it. 🤣